src/templates/header/index.hbs

```diff
@@ -2,7 +2,7 @@
 
 {{> skip-to }}
 
-<header class="c-header {{headerCssClassModifier}}" data-sticky-element>
```

Comment:
Should you not remove this setting from the docs json file too / add the new one to that now`useTransparentHeader`?

Reply:
Done